repo

```diff
@@ -4,9 +4,9 @@ upstream: "https://github.com/ocaml/opam-repository/tree/master/"
 announce: [
 """
 [WARNING] opam is out-of-date. Please consider updating it (https://opam.ocaml.org/doc/Install.html)
-""" {(opam-version >= "2.1.0~~" & opam-version < "2.1.6") | opam-version < "2.0.10" | (opam-version >= "2.2.0~~" & opam-version < "2.2.0")}
+""" {(opam-version >= "2.1.0~~" & opam-version < "2.1.6") | opam-version < "2.0.10" | (opam-version >= "2.2.0~~" & opam-version < "2.2.1")}
 """
-[INFO] opam 2.1 and 2.2 include many performance improvements over 2.0; please consider upgrading (https://opam.ocaml.org/doc/Install.html)
+[INFO] opam 2.1 and 2.2 include many performance and security improvements over 2.0; please consider upgrading (https://opam.ocaml.org/doc/Install.html)
```

Comment:
```suggestion
[INFO] opam 2.1 and 2.2 include many performance and security improvements over %{opam-version}%; please consider upgrading (https://opam.ocaml.org/doc/Install.html)
```
(similarly; I double-checked that opam 2.0.0 expands variables in post-messages!)